docs/developer/architecture/core/uisettings-service.asciidoc

```diff
@@ -38,3 +92,66 @@ export class MyPlugin implements Plugin {
 }
 
 ----
+
+Registeration only requires a schema for validation on read and write, all {kib-repo}blob/{branch}/docs/development/core/server/kibana-plugin-core-server.uisettingsparams.md[other parameters] are optional.
+
+See:
```

Comment:
I read somewhere that we use `Refer` instead of `See` for accessibility

Reply:
Better yet, I added links to the API docs directly from the relevant text 😉